schema/new_library_verification_cases.schema.json

```diff
@@ -0,0 +1,1062 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "properties": {
+        "cases": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "no": {
+                        "type": "integer",
+                        "minimum": 1
+                    },
+                    "run_simulation": {
+                        "type": "boolean"
+                    },
+                    "simulation_IO": {
+                        "type": "object",
+                        "properties": {
+                            "idf": {
+                                "type": "string",
+                                "pattern": "\\.idf$"
+                            },
+                            "idd": {
+                                "type": "string",
+                                "pattern": "\\.idd$"
+                            },
+                            "weather": {
+                                "type": "string",
+                                "pattern": "\\.epw$"
+                            },
+                            "output": {
+                                "type": "string",
+                                "pattern": "\\.csv$"
+                            },
+                            "ep_path": {
+                                "type": "string",
+                                "pattern": "\\.exe$"
+                            }
+                        },
+                        "required": [
+                            "idf",
+                            "output"
+                        ],
+                        "additionalProperties": false
+                    },
+                    "expected_result": {
+                        "enum": [
+                            "pass",
+                            "fail"
+                        ]
```

Comment:
@leijerry888 - Could we get rid of these here and in general in all verification case files since they're already in the library?